feature-libs/checkout/components/components/shipping-address/shipping-address.component.html

```diff
@@ -30,14 +30,14 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
         >
           <div
             class="cx-shipping-address-card-inner"
-            (click)="selectAddress(card.address)"
+            (click)="onAddressCardSelect(card.address)"
```

Comment:
just curious to know why a new function was added instead of modifying the existing `selectAddress` to render a global message? 

if the new function is needed, it will also require a test in the in the spec file.

Reply:
The original function gets called onInit so I wrote a new one so that the message only appears if the button was pressed